### PR TITLE
feat: Update position of dropdown in Navbar component for smaller screens

### PR DESCRIPTION
**Pull request by Ratul-Saha-003**
I have updated the position of the dropdowns in the Navbar component.
**Before**:
Screenshot (81)

**After**:
Screenshot from 2024-03-13 19-11-20

This Pull Request **resolves** issue #139

### CODE REVIEW

File: src/jio-navbar.css [hunk 1]
```diff
@@ -208,6 +208,11 @@ button.nav-link {
     order: 5;
   }
 
+  .dropdown{
```
Comment (halkeye) [inline src/jio-navbar.css:211]: Can you add a space between selector and bracket to match the rest? I'm surprised there's no linting error.

Comment (Ratul-Saha-003) [reply src/jio-navbar.css]: @halkeye Done.

**Comment by Ratul-Saha-003** [comment]
@halkeye I have made the necessary changes

**Comment by krisstern** [comment]
Fixed in #140

**Pull request closed without merging**